runtime_error.go

```diff
@@ -37,15 +37,15 @@ func (err RuntimeError) Error() string {
 
 // The stack
 
-// TraceFrame is tracing information about a single frame of the call stack.
-// TODO(sbarzowski) the difference from TraceElement. Do we even need this?
-type TraceFrame struct {
+// traceFrame is tracing information about a single frame of the call stack.
+// TODO(sbarzowski) the difference from traceElement. Do we even need this?
+type traceFrame struct {
 	Loc  ast.LocationRange
 	Name string
 }
 
-func traceElementToTraceFrame(trace TraceElement) TraceFrame {
-	tf := TraceFrame{Loc: *trace.loc}
+func traceElementTotraceFrame(trace traceElement) traceFrame {
```

Comment:
That should still be "To**T**raceFrame," even if the type's first letter is no longer capitalized.